tests/unit/test_utils.py

```diff
@@ -48,9 +48,9 @@ class TestGetColmValsAsFloats:
     def test_basic():
         """Check that function works as expected on non-problematic input"""
         # Arrange
-        rows = [["1", "2"], ["4", "5"]]
+        rows = [["1", "2"], ["4", "-5.5"]]
         colm_index = 1
-        expected_output = [2, 5]
+        expected_output = [2, -5.5]
```

Comment:
Why are we testing a negative number? I would think that the expected outputs would all be positive.